### PR TITLE
Completed node notes cli

### PR DESCRIPTION
**Pull request by tangm28**
Completed node notes cli
node-notes-cli-michael-tang

### CODE REVIEW

File: node-notes-cli/app.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const file = './data.json';
+const data = require(file);
+const fs = require('fs');
+const action = process.argv[2];
+const input = process.argv[3];
+const update = process.argv[4];
+
+fs.writeFile(file, JSON.stringify(data, null, 2), err => {
```
Comment (scbowler) [inline node-notes-cli/app.js]: You don't need this outer `writeFile`, you only need to update the file after there has been changes to the data object.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just 1 update needed. Let me know if you have any questions.

File: node-notes-cli/app.js (hunk 1 above)
Comment (tangm28) [reply node-notes-cli/app.js]: fixed!

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**